tests/openwisp2/sample_radius/migrations/0003_default_groups_and_permissions.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
     dependencies = [
         swapper.dependency('openwisp_radius', 'RadiusToken'),
         (model_app_label, '0002_initial_openwisp_app'),
-        (users_model_app_label, '0002_default_groups_and_permissions'),
+        (users_model_app_label, '0004_default_groups'),
```

Comment:
I suggest changing this code to use `swapper.dependency`, since it basically does the same thing.

eg:
```python
swapper.dependency(
    *swapper.split(rad_model), version='0002_initial_openwisp_app'
)
```

And the same for users.